mozetl/graphics/graphics_telemetry_dashboard.py

```diff
@@ -1061,7 +1042,6 @@ def advanced_layers_map(rdd):
         results = {
             "count": subset.count(),
             "compositors": subset.map(lambda p: (get_compositor(p),)).countByKey(),
-            "plugin_models": aggregate_plugin_models(subset),
```

Comment:
This field is still used in the code that generates the graphs https://github.com/FirefoxGraphics/telemetry/blob/bb0cc53a61a30acb22b3973f32f1721c572d02bc/www/chart-graphs.js#L363

So we need to either remove it from the other repo or hardcode a value.  I would go with the quick solution of hardcoding a value since this isn't a high priority dashboard.

[The current json](https://storage.googleapis.com/moz-fx-data-static-websit-8565-analysis-output/gfx/telemetry-data/windows-features.json) has `[]` in one of the values and I think it just [renders nothing](https://firefoxgraphics.github.io/telemetry/#view=windows-features&winver=6.2.0) so an empty list should work